packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
Consolidated the executors to this file.

Reply:
Can we move the tests for the `LibraryCommandletExecutor` over as well?

Reply:
No because they are integration tests. I've been trying to convert them into unit tests like some of the others in this package but the telemetry service dependency is making it very difficult. The telemetry service tests themselves are skipped in this package. We need to have a larger discussion about that module because it can cause a lot of problems with these refactors